test/test.js

```diff
@@ -83,27 +83,27 @@ describe('Metrics constructor', () => {
     it('should accept "webextension"', () => {
       beforeWebExt(global);
 
-      const m = new Metrics({ id: '@my-addon', version: '1.0.2', uid: 123, type: 'webextension' });
+      new Metrics({ id: '@my-addon', version: '1.0.2', uid: 123, type: 'webextension' });
 
```

Comment:
actually, I think these tests are wrong - they should verify m.type, as is done in the first test on line 79.

would you mind adding the missing asserts?